airflow/providers/openlineage/plugins/adapter.py

```diff
@@ -181,6 +188,7 @@ def start_task(
             ),
             job=self._build_job(
                 job_name=job_name,
+                job_type=_JOB_TYPE_TASK,
```

Comment:
Looks it is breaking change. Can't we support with job_type and without it?

Reply:
The Job type is just an additional facet that is supported in the [OL specs](https://openlineage.io/docs/spec/facets/job-facets/job-type/#) I don't think it's a breaking change but just a nice to have that simplify the backend reading OL events to identify where the events coming from.

Reply:
I also don't consider this breaking change. In OL spec facets are atomic pieces of metadata and backend rather shouldn't fail on additional facets attached.

Reply:
Agreed with @JDarDagran - additions are not breaking changes in OpenLineage contract. Consumers are encouraged to ignore facets they don't understand rather than rejecting the event.